queries.sql

```diff
@@ -1,7 +1,18 @@
 -- Part 1: Test it with SQL
+id INT
+employer VARCHAR
+name VARCHAR
+skills VARCHAR
 
 -- Part 2: Test it with SQL
+SELECT name
+FROM employer
+where location = "St. Louis";
```

Comment:
```suggestion
WHERE location = "St. Louis City";
```

Your query is pretty close to being correct. Though it doesn't affect the output, you can make where uppercase to match the other commands. For this to pass, you'll need to search for St. Louis City instead of St. Louis.